directory_sso_api_client/backends.py

```diff
@@ -31,10 +32,13 @@ def authenticate(self, request):
     def get_user(self, session_id):
         response = sso_api_client.user.get_session_user(session_id)
         response.raise_for_status()
-        parsed = response.json()
-        return models.SSOUser(
+        return self.build_user(session_id=session_id, parsed=response.json())
+
+    def build_user(self, session_id, parsed):
+        SSOUser = auth.get_user_model()
+        return SSOUser(
             pk=parsed['id'],
             email=parsed['email'],
```

Comment:
This assumes that the email is going to be the username field, which may not always be true for a custom user model.
Would you consider using https://docs.djangoproject.com/en/2.2/topics/auth/customizing/#django.contrib.auth.models.CustomUser.USERNAME_FIELD ?